src/runtime/error.ts

```diff
@@ -5,6 +5,7 @@ import { Trace } from "./trace.ts";
 
 export interface RuntimeErrorOptions extends ErrorOptions {
 	meta?: any;
+	statusCode?: number;
```

Comment:
i don't think we still need status code here since this was built specifically for routes, right?

Reply:
nvm resolved over vc